parcels/field.py

```diff
@@ -1077,6 +1067,12 @@ def time_index(self, time):
         else:
             return (time_index.argmin() - 1 if time_index.any() else 0, 0)
 
+    def __getitem__(self, key):
+        if _isParticle(key):  # Only works if not isinstance(key, SciPyParticle) [e.g. BaseParticleAccessor]
```

Comment:
I think that stray comment needs to go - obviously it's working now. (Though that can be because of the new Accessor I designed.)

Essentially: if the check fails, you get an infinite-loop recursion, because it unpacks the `key` by-attribute - the last attribute is the `ScipyParticle` - which this function than unpacks and attempts to unravel again - rense-and-repeat to infinity.